README.md

```diff
@@ -61,22 +61,14 @@ System.setProperty("jenkins.model.DirectoryBrowserSupport.CSP", "default-src 'se
 Apply and restart the Jenkins server. All assets should now be served correctly.
 
 ### Command-line
-Install the [Allure command-line tool](https://www.npmjs.com/package/allure-commandline), and process the results directory:
```

Comment:
for what you delete this?
it helps end users in there own projects setup and use allure reporter and allure report engine.


If you want to describe how generate reports for wdio-allure-reporter "unit" tests you should update his part of README https://github.com/webdriverio/wdio-allure-reporter/blob/master/README.md#development